### PR TITLE
Feature/new frontend

### LINKED ISSUE #95: Add senario  information to landing page
Edit the current landing page to contain the information:
- [x] Add scenerio story for each of the teams,
- [x] Networks - All networks for the blue team, only the first one for the red team
- [x] GDRP - Last step in the signup process to be a box to tick about GDPR

### PR DESCRIPTION
**Pull request by MulvadT**

### CODE REVIEW

File: frontend/privateOLD/wireguard.conf.tmpl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Interface]
```
Comment (rasmim) [inline frontend/privateOLD/wireguard.conf.tmpl]: how is this used in the code?

File: database/gameUser.go [hunk 2]
```diff
@@ -15,16 +15,18 @@ type Team string
 const (
 	RedTeam  Team = "red"
 	BlueTeam Team = "blue"
+	NoTeam   Team = "no"
```
Comment (rasmim) [inline database/gameUser.go:18]: Is this value used in the code?

File: database/gameUser.go [hunk 3]
```diff
@@ -37,12 +39,37 @@ func AddUser(ctx context.Context, username, email, password, gameID string, team
 	user.Team = team
 	user.CreatedAt = time.Now()
 	user.GameID = gameID
+	user.JoinedGame = false
```
Comment (rasmim) [inline database/gameUser.go:42]: let's discuss this part, I do not get why we need this line.

**Review by rasmim — COMMENTED** [review]
There are some things that we should discuss.

File: frontend/privateOLD/wireguard.conf.tmpl (hunk 1 above)
Comment (MulvadT) [reply frontend/privateOLD/wireguard.conf.tmpl]: This file is not used as all files in privateOLD, but wireguard.conf.tmpl is also in the new folder 

The file are used here
https://github.com/aau-network-security/defatt/blob/2a85617eafd083a4f1387bd5689b6d31c5d6d293/frontend/handler.go#L401

Used to make the VPN config for the user.

**Review by rasmim — APPROVED** [review]

**Comment by rasmim** [comment]
The interface has a bug in the RED WG configuration file. Sometimes it gets the correct IPs in the AllowedIP, other times is just getting an IP they should not. Check the attached photo configurations:

https://imgur.com/a/teo5MVL

Moreover, the timer has another bug. It displays a time for  Red and a time for Blue users. It appears problem when multiple users are registering. 

@MulvadT

**Pull request merged**